src/Simulation/Simulation.jl

```diff
@@ -681,23 +611,45 @@ function _calculate_potential!( sim::Simulation{T, CS}, potential_type::UnionAll
         elseif length(sor_consts) > 1 && CS == Cartesian
             sor_consts = T(sor_consts[1])
         end
-        if ismissing(refinement_limits)
-            refinement_limits::NTuple{3, T} = CS == Cylindrical ? (T(1e-5), T(1e-5) / (0.25 * grid.axes[1][end]), T(1e-5)) : (T(1e-5), T(1e-5), T(1e-5))
-        end
-        if ismissing(min_grid_spacing)
-            min_grid_spacing::NTuple{3, T} = CS == Cylindrical ? (T(1e-5), T(1e-5) / (0.25 * grid.axes[1][end]), T(1e-5)) : (T(1e-5), T(1e-5), T(1e-5))
+        min_tick_distance::NTuple{3, T} = if CS == Cylindrical
+            if !ismissing(min_tick_distance)
+                if min_tick_distance isa length_unit
+                    world_r_mid = (sim.world.intervals[1].right + sim.world.intervals[1].left)/2
+                    min_distance_z = min_distance_r = T(to_internal_units(internal_length_unit, min_tick_distance))
+                    min_distance_r, min_distance_z / world_r_mid, min_distance_z
+                else 
+                    T(to_internal_units(internal_length_unit, min_tick_distance[1])),
+                    T(to_internal_units(internal_angle_unit,  min_tick_distance[2])),
+                    T(to_internal_units(internal_length_unit, min_tick_distance[3]))
+                end 
+            else
+                (T(1e-5), T(1e-5) / (0.25 * grid.axes[1][end]), T(1e-5)) 
```

Comment:
Are these default values documented somewhere (docs?)

Reply:
Now they are. In the docstrings: b7faa70